src/renderer/modals/Send/index.js

```diff
@@ -22,6 +25,11 @@ class SendModal extends PureComponent<{}, { stepId: StepId }> {
 
     const isModalLocked = ["recipient", "confirmation"].includes(stepId);
 
+    const rest = {};
```

Comment:
Does it have to be rest obj if we only have use-case of onClose for now?

Reply:
i did that because the Modal has a default for onClose param that is set to noop.
previously i did directly onClose = {this.props.onClose || noop} but val did not like.
So i did that